Middleware/DoctrineFlushMiddleware.php

```diff
@@ -5,12 +5,16 @@
 namespace HandcraftedInTheAlps\Bundle\SuluResourceBundle\Middleware;
 
 use Doctrine\ORM\EntityManagerInterface;
+use HandcraftedInTheAlps\Bundle\SuluResourceBundle\Common\DisableInterface;
+use HandcraftedInTheAlps\Bundle\SuluResourceBundle\Common\DisableTrait;
 use Symfony\Component\Messenger\Envelope;
 use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
 use Symfony\Component\Messenger\Middleware\StackInterface;
 
-class DoctrineFlushMiddleware implements MiddlewareInterface
+class DoctrineFlushMiddleware implements MiddlewareInterface, DisableInterface
 {
+    use DisableTrait;
```

Comment:
thats basically agains how services are defined.

Services itself should never have a state and so should not have setter or getters.

Reply:
I need to be able to disable services during an import. Discussed that with @wachterjohannes and @nnatter a few days ago, we decided to solve it like that. Do you have another approach to disable a service?

Reply:
@luca-rath @wachterjohannes @nnatter We should create a `DisableFlushStamp` which we add to the dispatch function in your import e.g.:

https://symfony.com/doc/current/components/messenger.html#adding-metadata-to-messages-envelopes

and check for that stamp in the flush middleware.

Reply:
@alexander-schranz But we need the same functionality also for more services, that are not related with messenger. e.g. data mappers, mergers, ...

Reply:
Then I would say the messages should not be used in the import at all and only the services which are needed should be called in the import and not use messages at all.

Reply:
We need to use messages, because we don't know if the import will be synchronous or asynchronous. Both should be possible.

Reply:
using a `DisableFlushStamp` for the messenger sounds good. i still think that it would be okay to disable the `ImportDataMapper` per default and allow to enable it, but we can discuss that somewhere else 🙂

Reply:
@nnatter still nothing I would add to this library.

Reply:
well i think the `DisableFlushStamp` makes sense here 🙂